templates/guides/get-started.md

```diff
@@ -0,0 +1,216 @@
+---
+page_title: "Get started with Astro Terraform Provider"
+---
+
+# Get started with Astro Terraform Provider
+In this guide, we will automate the onboarding of a new team by creating and managing an Astro Workspace and Deployment. By the end of this tutorial, you will have a fully automated setup that is reproducible and easily scalable to more teams.
+
+## Step 1: Create Your Terraform Working Directory
+1. Create a folder, `my-data-platform` for your Terraform project.
+2. Save the following code in a file named `terraform.tf`:
+```
+terraform {
+  required_providers {
+    astro = {
+      source  = "astronomer/astro"
+      version = "1.0.0"
+    }
+  }
+}
+
+provider "astro" {
+  organization_id = &lt;your-organization-id&gt;
+}
+```
+3. Insert your organization's ID for `<your-organization-id>`. The working directory will contain all your Terraform code, and all Terraform commands will be run from this directory.
+
+## Step 2: Initialize the Terraform Working Directory
+1. Run `terraform init`. You will see Terraform downloading and installing the Astro Terraform provider to your local computer:
+```
+$ terraform init
+
+Initializing the backend...
+
+Initializing provider plugins...
+- Finding astronomer/astro versions matching "1.0.0"...
+- Installing astronomer/astro v1.0.0...
+- Installed astronomer/astro v1.0.0 (signed by a HashiCorp partner, key ID F5206453FDEA33CF)
+
+...
+
+Terraform has been successfully initialized!
+
+You may now begin working with Terraform. Try running "terraform plan" to see
+any changes that are required for your infrastructure. All Terraform commands
+should now work.
+```
+2. The versions and hashes of providers are stored in a generated file `.terraform.lock.hcl`. Store this file in version control.
```

Comment:
```suggestion
2. The versions and hashes of providers are stored in a generated file `.terraform.lock.hcl`. Store this file in your version control system.
```

>Store this file in your version control system. 

this might be better as a tip after the step - since for a get started, users might not keep this example or project, so it wouldn't exactly require version control, but it's a best practice for production development.

Reply:
Good point, I'll make that recommendation into a note